src/main/java/org/glassfish/copyright/AbstractCopyright.java

```diff
@@ -93,6 +95,8 @@
     private static Pattern bsdpat = Pattern.compile(
 	"(THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS)"+
 	"|(SPDX-License-Identifier: BSD-3-Clause)", Pattern.MULTILINE);
+	// patter to detect DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+	protected static Pattern dontpat = Pattern.compile(DONT_ALTER_HEADER);
```

Comment:
Personally, I'd make this variable name a little easier to understand and readable.  I know you were just following the pattern for the "bsdpat" variable, but I would think something like "doNotAlterPattern" would be clearer.  I'll leave it to you though.

Reply:
fixed in 620fdb2